metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -255,7 +255,10 @@ object ServerCommands {
        |
        |Note: requires 'metals/inputBox' capability from language client.
        |""".stripMargin,
-    "[string], where the string is a directory location for the new file."
+    """|[string[]], where the first is a directory location for the new file.
```

Comment:
We actually had this just as a string before, when in reality this as an Array with a URI in the first position and a filename in the second position.